### ISSUE TITLE
Cannot join game.

### ISSUE THREAD

**Issue by Risto-Paasivirta** [ValveSoftware/Dota2-Gameplay#13251]
### Description

Cannot connect any ranked match.
Get kicked out and penalized because the system doesn't work.
Tried restarting and even re-installing Dota2.

**Comment by vivuor** [comment]
Yep same. Finland Steam connection manager is N/A. Many people who post about this is issue are from Finland.

**Comment by jaakkouu** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/15669666/a48ed97d-3e63-4e1f-a974-70782ef76f15)

Why you do this Gabe :(

**Comment by badumbl** [comment]
Same, from Estonia

**Comment by vivuor** [comment]
> Same, from Estonia

Yes. Noticed this also in Reddit when another user from Estonia has this problem. Some of the ss from here are from Russia (client languange). So this probably affects places near Helsinki Steam connection manager and are using the same or something. I just hope Valve fixes this soon.

**Comment by osk4r99** [comment]
I only tested it in bot game since I did not want to get another ban. But I was only unable to connect to EU-West server on co-op bots but the other servers where fine (Eu-east and Russia). I don't know if it will behave differently in normal game or if it was a fluke.

**Comment by 4Z4THOTH** [comment]
im from estonia,have same problem

**Comment by Ro0pE** [comment]
If you dont want to wait Valve to fix the problem, I'm currently using VPN, changed my location from finland to denmark and its working like a charm now.

**Comment by vivuor** [comment]
> If you dont want to wait Valve to fix the problem, I'm currently using VPN, changed my location from finland to denmark and its working like a charm now.

That is smart. I found out that its fine if I play in eu east and russian.

**Comment by tomkivi** [comment]
EU west still not working from Finland. Having to queue in to EU East. Please fix it already, so I get more English speaking players in my games.

**Comment by Ro0pE** [comment]
After few days with VPN I just rebooted My router and now Im able to play EU West without VPN. Game is much more fun when you can read and understand your team chat... 😄
